### ISSUE TITLE
Update Zero Day Labs logo on Website

### ISSUE THREAD

**Issue by DBees** [openhwgroup/openhwgroup.org#590]
Please update the Zero Day Labs Logo on the website

The new logo is [here](https://drive.google.com/file/d/18sv87xKNAYS-usYqWefqrgalVMz_h_Pp/view?usp=sharing) as was shared by @sandro2pinto

thank you
Duncan

**Comment by autumnfound** [comment]
Hi, here is the preview of the changes @DBees 

https://preview-591--openhwgroup-org.eclipsecontent.org/
https://preview-591--openhwgroup-org.eclipsecontent.org/membership/members/

Let me know if this is good to go live!

**Comment by DBees** [comment]
@autumnfound I get this message in Chrome when clicking your preview link 

![image](https://user-images.githubusercontent.com/8946246/223167583-50677f44-1e58-4530-b6fb-eeafce77f727.png)

**Comment by DBees** [comment]
But in any case @autumnfound the preview looks fine on Edge. thanks!

**Comment by autumnfound** [comment]
Yeah, that's something we were informed of last week. It's a good new feature in Chrome that's meant to help people avoid putting their data on fake sites. The preview sites are intended to look like the original site and it was created by us, so it's safe to click through in this case!
